data/competencies.yml

```
@@ -307,6 +297,16 @@
   area: leadership
   domain: null
 
+- id: 82a8460f-8554-4837-9799-01174ea5be6f
+  summary: Has worked with teams outside of their home group (where home group will be one of Customer Products, FT Core, Internal Products, Operations and Reliability,  Enterprise Services and Security, FT Group Products or FT Labs)
```

Fixes a double space
```suggestion
  summary: Has worked with teams outside of their home group (where home group will be one of Customer Products, FT Core, Internal Products, Operations and Reliability, Enterprise Services and Security, FT Group Products or FT Labs)
```

---

How do you feel about not listing the groups? It'll only get out of date and if line managers are informed about what a Group is (which I don't think it's too optimistic as TLG are being urged, and urging others, to use the term 'Group' more), then it's fairly superfluous. Can always add the list back if it's a constant cause of confusion

---

I added the term because "Group" felt very ambiguous in this situation.